transport/http/transport_test.go

```diff
@@ -297,3 +297,11 @@ func TestTransportClientWithMaxIdleConnections(t *testing.T) {
 
 	assert.NotNil(t, transport.client)
 }
+
+func TestTransportClientWithSocks5Proxy(t *testing.T) {
+	// Unfortunately the client.transport.Dialer is obfuscated in the client, so we can't really
```

Comment:
can we run a small socks5 proxy (from googling: https://github.com/armon/go-socks5) and have a request proxied to a dispatcher through the socks5 proxy?